src/make-http-request.coffee

```diff
@@ -2,8 +2,14 @@ request = require 'superagent'
 coreUrl = require 'url'
 corePath = require 'path'
 
+DEFAULT_HEADERS = require './default-headers'
+
 request = request.agent() if request.agent?
 
+# Superagent will only auto-parse responses from Content-Types it recognizies
+# Add the Content-Type in use by the JSON API spec
+request.parse[DEFAULT_HEADERS['Content-Type']] = JSON.parse
```

Comment:
This should use the Accept header, since that's the one that sets type it gets back from the server. Seems needlessly pedantic here, but JSON-API allows you to set the content type as application/json or application/json+patch for certain requests.